packages/next-app/src/utils/hardcodedData.ts

```diff
@@ -1,6 +1,57 @@
+import { Chain } from "wagmi";
+
 import { ActionAttribute } from "#/contexts/AdminToolsContext";
+import gaugeGql from "#/lib/gaugesGql";
+import poolGql from "#/lib/gql";
+
+const inputField =  {
+  label: "Pool ID",
+  key: "poolId",
+  placeholder: "Insert the Pool ID here",
+  type: "string",
```

Comment:
I think we should remove this if this is not being used to avoid WTFs when looking at the code